### ISSUE TITLE
Doesn't handle input with only one column, at least with {objectMode: true, columns: true}

### ISSUE THREAD

**Issue by denishaskin** [klaemo/csv-stream#17]
Sample code:

```
var str = require('string-to-stream'), csv = require('csv-streamify');

str("COL0,COL1\ncol0,col1\n").pipe(csv({objectMode: true, columns: true})).on('data', function(chunk) {
    console.log(chunk);
});

str("COL0\ncol0\n").pipe(csv({objectMode: true, columns: true})).on('data', function(chunk) {
    console.log(chunk);
}).on('error', function(error) {
    console.log(error);
});
```

Expected output:

```
{ COL0: 'col0', COL1: 'col1' }
{ COL0: 'col0' }
```

Actual output:
```
{ COL0: 'col0', COL1: 'col1' }
```
(maybe with an error?)

Something happens when input has only one column.

**Comment by klaemo** [comment]
Thanks for reporting this issue. I can reproduce it and will try to fix it asap.

**Comment by klaemo** [comment]
Fix published as 3.0.1.

**Comment by denishaskin** [comment]
:+1:
